tests/unit/lms/services/jwt_test.py

```diff
@@ -152,6 +164,22 @@ def svc(self, lti_registration_service, rsa_key_service):
         return svc
 
 
+class Test_RequestsPyJWKClient:
```

Comment:
I think `_RequestsPyJWKClient` should be private but also tested independently.

Here we test our changes to the class and the test that use the mocked version tests how it all plays together.